bot/seasons/easter/easter_riddle.py

```diff
@@ -0,0 +1,95 @@
+import asyncio
+import logging
+import random
+from json import load
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+
+from bot.constants import Colours
+
+log = logging.getLogger(__name__)
+
+with open(Path('bot', 'resources', 'easter', 'easter_riddle.json'), 'r', encoding="utf8") as f:
+    RIDDLE_QUESTIONS = load(f)
+
+TIMELIMIT = 10
+
+
+class EasterRiddle(commands.Cog):
+    """This cog contains the command for the Easter quiz!"""
+
+    def __init__(self, bot):
+        self.bot = bot
+        self.winners = []
+        self.correct = ""
+        self.current_channel = None
+
+    @commands.command(aliases=["riddlemethis", "riddleme"])
+    async def riddle(self, ctx):
+        """Gives a random riddle questions, then provides 2 hints at 10 second intervals before revealing the answer"""
+        if not self.current_channel:
+            self.current_channel = ctx.message.channel
+
+            random_question = random.choice(RIDDLE_QUESTIONS)
+            question = random_question["question"]
+            hints = random_question["riddles"]
+            self.correct = random_question["correct_answer"]
+
+            description = f"You have {TIMELIMIT} seconds before the first hint.\n\n"
+
+            q_embed = discord.Embed(title=question, description=description, colour=Colours.pink)
```

Comment:
how about `riddle_embed` and `hint_embed` instead of these awful variable names?